pkg/cmd/dev/test.go

```diff
@@ -257,6 +257,26 @@ func (d *dev) test(cmd *cobra.Command, commandLine []string) error {
 		testTargets = append(testTargets, target)
 	}
 
+	// List of bazel integration tests that will fail when running `dev test pkg/...`
+	var integrationTests = map[string][2]string{
+		"pkg/acceptance":              {"acceptance_test", "`dev acceptance`"},
+		"pkg/compose":                 {"compose_test", "`dev compose`"},
+		"pkg/compose/compare/compare": {"compare_test", "`dev compose`"},
+		"pkg/testutils/docker":        {"docker_test", "N/A: docker_test has no dev command corresponding to it"},
```

Comment:
Combined with the logic below, the error message will look like: "To run this test, run N/A: docker_test has no dev command corresponding to it", which is very confusing. Instead, I would say leave the string empty, and then handle that case gracefully below.